sdk/file_sdk/UnitTests/zip_ut/zip_tests.cpp

```diff
@@ -0,0 +1,71 @@
+#include <QString>
+#include <QtTest>
+#include <Qt>
+#undef signals
```

Comment:
why in the middle of includes?

Reply:
because QT is defining signals which is a name of member in libGsf.

---
In reply to: [139612073](https://github.com/AzureAD/rms-sdk-for-cpp/pull/158#discussion_r139612073) [](ancestors = 139612073)